__tests__/nested-test.js

```diff
@@ -19,6 +19,24 @@ describe('resolve.nested', function () {
 
     expect(nested({ column })(rowData)).toEqual(expected);
   });
+  it('resolves nested values, using a custom getter', function () {
+    const lastName = 'demo';
+    const property = a => (a.name || {}).last;
+    const rowData = {
+      name: {
+        last: lastName
```

Comment:
Might as well change naming so you get only `last`. So just `const last = 'demo';`.